### ISSUE TITLE
V2.22.9 with ESP32 Dev module, WiFi, GPS and Compass can't be compiled

### ISSUE THREAD

**Issue by GauchoFPV** [zs6buj/AntTracker#86]
Hi Eric,
I tried to compile the latest version but got several errors:

![image](https://github.com/user-attachments/assets/ca6676b0-2488-4b05-8fd9-011e72a77a12)
![image](https://github.com/user-attachments/assets/1d301f1e-8315-416d-894f-63552974d289)

I could change WiFiUDP.h to WiFiUdp.h because I assume it got changed in the library but am not able to correct the other issues because I don't have any programming skills...

Could you help me please?
Here is my config.h:

[config.h.txt](https://github.com/user-attachments/files/17160094/config.h.txt)

Thank you and have a nice day

**Comment by zs6buj** [comment]
GauchoFPV: Thanks for the library name fix. Updated in v2.22.11.

In your config.h you have:

```
    #define SSD1306_DISPLAY         // OLED display type
    /* Below please choose either Touch pin-pair or Digital pin-pair for display scrolling
      *  Pin == -1 means the pin-pair is not used
      */
    #define Pup           -1        // Board Button 1 to scroll the display up
    #define Pdn           -1        // Board Button 2 to scroll the display down
```
If you #define SSD1306_DISPLAY, then you must define pageup(Pup) and pagedown(Pdn) gpio pins to scroll the display. I have used your config setup in v2.22.11. and I'm getting a clean compile here now.

**Comment by GauchoFPV** [comment]
Hi Eric,

thank you very much and shame on me!

I just found another small mistake in the antTrack.ino, Line 137 : #include <globals.h> --> #include "globals.h"

Now I can compile and upload it without  problems, thank you so much!!!

**Comment by zs6buj** [comment]
Hi, It's best to use v2.22.11 if you can. But also v2.22.09 has globals.h included in the antTrack folder.
![image](https://github.com/user-attachments/assets/ed5d4956-26b1-4e01-9c63-acd9e33fb417)

**Comment by GauchoFPV** [comment]
Of course, I used 2.22.11 and globals.h was in the folder but Ibhad to replace  the <> with "" to make it work.

**Comment by zs6buj** [comment]
Can we close this issue now?

**Comment by GauchoFPV** [comment]
Sorry, I'm not used to GitHub but yes, the ticket can be closed. Thank you very much!!!
